packages/odyssey/src/scss/base/_typography-list.scss

```diff
@@ -42,3 +42,22 @@ ul:not([class]) {
 ol:not([class]) {
   list-style-type: decimal;
 }
+
+dl:not([class]) {
```

Comment:
This styling isn't really suited for key/value pairs, but I didn't want to start introducing HTML element variant styles. We've discussed List becoming a full component - I assume `dl` will make its way in there alongside `ul`, `ol`, and custom lists.

Reply:
Whoops, meant to ping you directly @alexdahl-okta - I know you raised some interest in this element being styled

Reply:
Looks great! We’ll probably work with doc writers to see how they’d like to use, if they have input on its style over time.

I can see value for key/value pairs in contexts like the Admin Console. For example, today the Identity Providers table expands rows and shows key/value pairs with a similar style to this. I'm not sure how much `dl` should go in that direction, but probably viable to use it in that way.